PowerUp/app/src/main/res/values/integers.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
     <integer name="max_progress">100</integer>
     <integer name="half_progress">50</integer>
+    <integer name="display_points_request_code">1</integer>
```

Comment:
How is this request code used and why is it defined as a resource? Use this file for integers you need to use in the layout file. If you need constants, define them in Java as enumerates or static final variables depending on what makes sense